persistence-modules/spring-hibernate-5/pom.xml

```diff
@@ -108,6 +108,26 @@
             <artifactId>h2</artifactId>
             <version>${h2.version}</version>
         </dependency>
+		<dependency>
+		    <groupId>jakarta.xml.bind</groupId>
+		    <artifactId>jakarta.xml.bind-api</artifactId>
+		    <version>2.3.2</version>
```

Comment:
Pls use properties for these dependencies.

Reply:
This has been resolved